### PR TITLE
Use HDP3 for suite-1 with JDK 17

### PR DESCRIPTION
**Pull request by wendigo**

### CODE REVIEW

**Comment by wendigo** [comment]
I can confirm that it was indeed run against JDK 17 for HDP3: 

```
2021-10-05T09:25:09.8640583Z presto-master       | INFO: Java version: 17
2021-10-05T09:25:10.0058466Z presto-worker       | INFO: Java version: 17
```

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -454,6 +454,9 @@ jobs:
           - config: default
             suite: suite-1
             jdk: zulu_17
+          - config: hdp3
```
Comment (losipiuk) [inline .github/workflows/ci.yml:457]: I would just keep `hdp3` for to keep CI queue small. 
@findepi WDYT?

**Review by losipiuk — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -475,7 +475,7 @@ jobs:
         run: |
           testing/bin/ptl suite run \
             --suite ${{ matrix.suite }} --config config-${{ matrix.config }} --bind=off --logs-dir logs/ --timeout 2h \
-            --trino-jdk-version zulu_${{ matrix.jdk }}
+            --trino-jdk-version ${{ matrix.jdk }}
```
Comment (findepi) [inline .github/workflows/ci.yml:481]: labels are longer on CI web UI. did we buy anything else with the change?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (findepi) [reply .github/workflows/ci.yml:457]: yep, replace default with hdp3

File: .github/workflows/ci.yml (hunk 2 above)
Comment (findepi) [reply .github/workflows/ci.yml:481]: (i would revert)

**Comment by wendigo** [comment]
@findepi replaced and dropped first commit :)

File: .github/workflows/ci.yml (hunk 1 above)
Comment (wendigo) [reply .github/workflows/ci.yml:454]: Replaced

**Comment by wendigo** [comment]
@losipiuk can you merge it please?

**Pull request merged**